pkg/controller/apiserver/apiserver_controller.go

```diff
@@ -252,11 +252,11 @@ func (r *ReconcileAPIServer) Reconcile(ctx context.Context, request reconcile.Re
 		r.status.SetDegraded(operatorv1.ResourceReadError, "Error querying installation", err, reqLogger)
 		return reconcile.Result{}, err
 	}
-	if variant == "" {
+	if installationSpec.Variant == "" {
 		r.status.SetDegraded(operatorv1.ResourceNotReady, "Waiting for Installation to be ready", nil, reqLogger)
```

Comment:
I think the message here needs to be changed to something like "Waiting for Installation Variant to be set". I think the ResourceNotReady is good though.

Reply:
Agreed, message updated